packages/rpc-core/src/rpc-methods/__tests__/send-transaction-test.ts

```diff
@@ -61,33 +59,33 @@ const MOCK_PKCS8_PRIVATE_KEY =
         0x30, // ASN.1 sequence tag
         0x2e, // Length of sequence (46 more bytes)
 
-            0x02, // ASN.1 integer tag
-            0x01, // Length of integer
-                0x00, // Version number
+        0x02, // ASN.1 integer tag
+        0x01, // Length of integer
+        0x00, // Version number
 
-            0x30, // ASN.1 sequence tag
-            0x05, // Length of sequence
-                0x06, // ASN.1 object identifier tag
-                0x03, // Length of object identifier
-                    // Edwards curve algorithms identifier https://oid-rep.orange-labs.fr/get/1.3.101.112
-                        0x2b, // iso(1) / identified-organization(3) (The first node is multiplied by the decimal 40 and the result is added to the value of the second node)
-                        0x65, // thawte(101)
-                    // Ed25519 identifier
-                        0x70, // id-Ed25519(112)
+        0x30, // ASN.1 sequence tag
+        0x05, // Length of sequence
+        0x06, // ASN.1 object identifier tag
+        0x03, // Length of object identifier
+        // Edwards curve algorithms identifier https://oid-rep.orange-labs.fr/get/1.3.101.112
+        0x2b, // iso(1) / identified-organization(3) (The first node is multiplied by the decimal 40 and the result is added to the value of the second node)
+        0x65, // thawte(101)
+        // Ed25519 identifier
+        0x70, // id-Ed25519(112)
 
         /**
          * Private key payload
          */
         0x04, // ASN.1 octet string tag
         0x22, // String length (34 more bytes)
 
-            // Private key bytes as octet string
-            0x04, // ASN.1 octet string tag
-            0x20, // String length (32 bytes)
-                16, 192, 67, 187, 170, 210, 152, 95,
-                180, 204, 123, 21, 81, 45, 171, 85,
-                188, 91, 164, 34, 8, 0, 244, 56,
-                209, 190, 255, 201, 212, 94, 45, 186,
+        // Private key bytes as octet string
+        0x04, // ASN.1 octet string tag
+        0x20, // String length (32 bytes)
+        16, 192, 67, 187, 170, 210, 152, 95,
+        180, 204, 123, 21, 81, 45, 171, 85,
+        188, 91, 164, 34, 8, 0, 244, 56,
+        209, 190, 255, 201, 212, 94, 45, 186,
```

Comment:
Just flagging this lint change in case you didn't mean to do that.

Reply:
Thanks, updated!